internal/kafka/command_link_describe.go

```diff
@@ -10,14 +10,14 @@ import (
 )
 
 type describeOut struct {
```

Comment:
I think we should rename this. Maybe something like `linkOut`, so it's not specific to one operation.

Reply:
done